### ISSUE TITLE
Legend ignore the "Percentage Decimals" setting

### ISSUE THREAD

**Issue by Jean-Daniel** [grafana/piechart-panel#118]
When displaying legend as percent, there is a settings to choose the number of decimals that defaults to 2.

Change it to anything else does not have any effect on the legend format.

**Comment by Ijin08** [comment]
I haven't had this issue except when setting decimals to 0. 
Do you refresh after changing decimal?

**Comment by Jean-Daniel** [comment]
You're right. I did first encounter the issue by setting 0 (as I don't need any decimal in my use case), and then did a quick test with other values but forget to refresh between each change.

**Comment by JBUCGitGuardian** [comment]
I had the problem on Grafana v7.4.5 (8a2c78d3f8).  
The decimal is working for any other value than 0 but doesn't work when you want no decimal.
